sources/north-america/us/oh/ODOT_TIMS_Road_Inventory.geojson

```diff
@@ -0,0 +1,1493 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "properties": {
+        "id": "ODOT_TIMS_Road_Inventory",
+        "name": "ODOT (Ohio) Road Inventory",
+        "description": "Ohio Department of Transportation (ODOT) - Transportation Information Mapping System (TIMS) Road Inventory published June 2024.",
+        "type": "tms",
+        "url": "https://tiles.mblaine.com/ohio/{zoom}/{x}/{y}.png",
+        "min_zoom": 0,
+        "max_zoom": 19,
+        "license_url": "https://wiki.openstreetmap.org/wiki/ODOT_TIMS#Right_to_Use",
+        "privacy_policy_url": "https://www.transportation.ohio.gov/help-center/privacy-notice-and-policies",
+        "permission_osm": "implicit",
+        "icon": "https://dam.assets.ohio.gov/image/upload/transportation.ohio.gov/assets/Logo_Assets/ODOT_Zephyr-HORZ-RGB.png",

```

Comment:
This icon will be shown in the corner of the editor’s map, possibly at full size. How about we use just [ODOT’s iconic zephyr logo](https://www.transportation.ohio.gov/wps/wcm/connect/gov/2ac66df0-55c6-4360-9f44-ff1435bbd068/favicon-ODOT.png?MOD=AJPERES) instead?